trunk/SUAVE/Methods/Noise/Fidelity_One/Propeller/propeller_noise_frequency.py

```diff
@@ -0,0 +1,327 @@
+                
+                #Phase shift due to offset or face alignment
+                phi_s = (2*m*blades_number*Mach_tip/Mach_section*convection_factor)*MCA[i]/propeller_diameter    
+        
+                
+                #Calculation of the Bessel function:
+                Arg    = m*blades_number*z*Mach_tip*np.sin(angle)/convection_factor #argument
+                NOrder = m*blades_number    #order
+                XJn    = scipy.special.jn(NOrder, Arg)
+                
+                if (kx==0):
+                    Gama_v = 2.0/3.0
+                    Gama_l = 1.0
+                    Gama_d = 1.0
+                else:
+                    Gama_v = (8.0/(kx*kx))*((2.0/kx)*np.sin(kx*0.5)-np.cos(kx*0.5))
+                    Gama_l = (2.0/kx)*np.sin(kx*0.5)
+                    Gama_d = (2.0/kx)*np.sin(kx*0.5)
+                
+                
+                pvm_factor = kx*kx*tb*Gama_v
+                pdm_factor = 1j*kx*(drag_coefficient[i]*0.5)*Gama_d
+                plm_factor = -1j*ky*(lift_coefficient[i]*0.5)*Gama_l
+                
+                const_integral = Mach_section*Mach_section*np.exp(1j*(phi_s+phi_zero))*XJn
+                
+                Pvm_section[i] = const_integral*pvm_factor 
+                Pdm_section[i] = const_integral*pdm_factor 
+                Plm_section[i] = const_integral*plm_factor
+                
+                #DIv_section[i] = XJn*Gama_v
+                #DIl_section[i] = XJn*Gama_l
+                            
+           #     print "SECTION = ", i, "Max_thick = ", max_thickness, "tb = ", tb, "Bd = ", Bd, "z = ",z, "  Mach section = ", Mach_section, "cl = ", lift_coefficient[i], "drag = ", drag_coefficient[i]
+               # print "z = ",z, "  Mach section = ", Mach_section
+         #       print "Phi0 = ", phi_zero, "  Phis = ", phi_s
+            
+            for i in range(1,number_sections-2):
+                I_pvm = I_pvm + Pvm_section[i]
+                I_pdm = I_pdm + Pdm_section[i]
+                I_plm = I_plm + Plm_section[i]
+            
+    #integration using trapezoidal rule       
+            h = (r0[number_sections-1]-r0[0])/number_sections 
+            Pvm = const*0.5*h*(Pvm_section[0]+Pvm_section[number_sections-1]+2.0*I_pvm)
+            Pdm = const*0.5*h*(Pdm_section[0]+Pdm_section[number_sections-1]+2.0*I_pdm)
+            Plm = const*0.5*h*(Plm_section[0]+Plm_section[number_sections-1]+2.0*I_plm)
+            
+            Pmb = (Pvm + Pdm + Plm)
+            
+            p_monopole = 2*np.real(Pvm*np.exp(-1j*m*blades_number*frequency_doppler*time))
+            p_drag = 2*np.real(Pdm*np.exp(-1j*m*blades_number*frequency_doppler*time))
+            p_lift = 2*np.real(Plm*np.exp(-1j*m*blades_number*frequency_doppler*time))
+            
+            p_total = 2*np.real(Pmb*np.exp(-1j*m*blades_number*frequency_doppler*time))
+            
+         #   print "monopole = ", p_monopole, "p_drag = ", p_drag, "p_lift = ", p_lift
```

Comment:
Remove comments

Reply:
I removed this file from this PR and will put it in a noise PR